website/content/docs/internals/telemetry/enable-telemetry.mdx

```diff
@@ -0,0 +1,93 @@
+---
+layout: docs
+page_title: Enable Vault telemetry
+description: >-
+  Step-by-step guide to enabling telemetry gathering with Vault
+---
+
+# Enable Vault telemetry gathering
+
+Collect telemetry data from your Vault installation.
+
+## Before you start
+
+- **You must have Vault 1.14 or later installed and running**.
+- **You must have access to your [Vault configuration](/vault/docs/configuration) file**.
+
+
+## Step 1: Choose an aggregation agent
+
+@include 'telemetry/supported-aggregation-agents.mdx'
+
+## Step 2: Enable at least one audit device
+
+To include audit-related metrics, you must enable auditing on at least device
+with the `vault audit enable` command. For example, to enable auditing for the
+`file` device and save the logs to `/var/log/vault_audit.log`:
+
+```shell-session
+$ vault audit enable file file_path=/var/log/vault_audit.log
+```
+
+By default, Vault replicates audit device configuration to all nodes within a
+cluster. Enterprise installations also replicate audit device configuration to
+the primary node and all secondaries in a cluster. To disable replication, use
+the `local` parameter:
+
+```shell-session
+$ vault audit enable file -local file_path=/var/log/vault_audit.log
+```
+
+
+## Step 3: Configure telemetry collection
+
+To configure telemetry collection, update the telemetry stanza in your Vault
+configuration with your collection preferences and aggregation agent details.
+
+For example, the following `telemetry` stanza configures Vault with the standard
```

Comment:
The prefix_filter isn't "standard telemetry defaults".

Reply:
That said, I wonder if it's worth mentioning in its own section in this page.  Many metrics solutions charge by the metric, so pointing out that it's possible to reduce the volume of metrics emitted by vault might be appreciated.

Reply:
Added a tip for filtering. Let me know if I'm misrepresenting anything